apps/roam/src/utils/migrateRelations.ts

```diff
@@ -13,50 +13,56 @@ const MIGRATION_PROP_NAME = "relation-migration";
 const migrateRelations = async (dryRun = false): Promise<number> => {
   const authorized = getSetting("use-reified-relations");
   if (!authorized) return 0;
-  const processed = new Set<string>();
-  const relationData = await getRelationData();
   let numProcessed = 0;
-  for (const rel of relationData) {
-    const key = `${rel.source}:${rel.relUid}:${rel.target}`;
-    if (processed.has(key)) continue;
-    processed.add(key);
-    if (!dryRun) {
-      const uid = (await createReifiedRelation({
-        sourceUid: rel.source,
-        destinationUid: rel.target,
-        relationBlockUid: rel.relUid,
-      }))!;
-      const sourceProps = getBlockProps(rel.source);
-      const dgDataOrig = sourceProps[DISCOURSE_GRAPH_PROP_NAME];
-      const dgData: Record<string, json> =
-        dgDataOrig !== null &&
-        typeof dgDataOrig === "object" &&
-        !Array.isArray(dgDataOrig)
-          ? dgDataOrig
-          : {};
-      const migrationDataOrig = dgData[MIGRATION_PROP_NAME];
-      let migrationData: Record<string, json> =
-        migrationDataOrig !== null &&
-        typeof migrationDataOrig === "object" &&
-        !Array.isArray(migrationDataOrig)
-          ? migrationDataOrig
-          : {};
-      if (migrationData[uid] !== undefined) {
-        console.debug(`reprocessed ${key}`);
+  await setSetting("use-reified-relations", false); // so queries use patterns
```

Comment:
How is this making it so that "queries use patterns"?  I don't see a reference to `use-reified-relations` in `getRelationData`